### ISSUE TITLE
WebScrobbler - Documentation Issue

### ISSUE THREAD

**Issue by undaunt** [FoxxMD/multi-scrobbler#111]
[Please check the FAQ](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/FAQ.md) before submitting a bug report.

**Describe the bug**
When attempting to follow the webscrobbler [documentation](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/configuration/configuration.md#webscrobbler) I was receiving no-configured sources and/or syntax errors.

**To Reproduce**
1. Create a configuration based, single user config from [here](https://github.com/FoxxMD/multi-scrobbler/blob/master/config/webscrobbler.json.example)
2. Attempt to start the app
3. Receive warnings about no configured source when playing.
4. Reconfigure based on the multi-user documentation with slugs.
5. Receive schema warnings and app does not fully load.

**Expected behavior**
The app should launch and not have schema errors, and receive webhooks from a valid source.

**Versions (please complete the following information):**
- multi-scrobbler: 0.6.1 on docker

**Additional context**
This appears to be resolved by updating the config to:

1. Not nesting the slug under **data** as shown in the multi-user example.
2. Including the type of **webscrobbler** (not noted in the docs, I saw it in the commit code

Here is my working config:

```
    {
      "type": "webscrobbler",
      "name": "WebScrobble",
      "slug": null,
      "clients": [],
      "data": {
        "whitelist": [],
        "blacklist": []
      }
    },
```

**Comment by TheFeelTrain** [comment]
In addition to this the environment variables like `WS_ENABLE` don't actually do anything. If you search for them in the code the only reference to them is in the documentation so as far as I can tell they never actually used. The entire WebScrobbler section needs updating.

**Comment by TheFeelTrain** [comment]
In addition to this the environment variables like WS_ENABLE don't actually do anything. If you search for them in the code the only reference to them is in the documentation. So as far as I can tell they never actually used. The entire WebScrobbler section needs updating.

**Comment by TheFeelTrain** [comment]
In addition to this the environment variables like WS_ENABLE don't do anything. If you search for any of them in the code, the only references made to them is in the documentation. So as far as I can tell they never actually get used. The entire WebScrobbler section needs updating.

**Comment by FoxxMD** [comment]
You are correct in that there was a discrepancy between what was shown in [webscrobbler.json](https://github.com/FoxxMD/multi-scrobbler/blob/master/config/webscrobbler.json.example?rgh-link-date=2023-11-30T19%3A09%3A04Z) and in the multi-user config, but either one should have worked without issue. `slug` should be in `data` but it is an optional property and should not cause schema warnings. I was not able to reproduce your issue when using MS setup with `config/webscrobbler.json`

Were you using the example data found in [webscrobbler.json](https://github.com/FoxxMD/multi-scrobbler/blob/master/config/webscrobbler.json.example?rgh-link-date=2023-11-30T19%3A09%3A04Z) but in `sources` in the [all-in-one config (`config.json`)](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/configuration/configuration.md#all-in-one-file-configuration)? This is the only area where `type` is required. If you were using `webscrobbler.json` directly in your config directory it is not necessary to include `type`.

[MS with All-in-one config](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/configuration/configuration.md#all-in-one-file-configuration)
```
/config
  --> config.json

# config.json
...
  "sources": [
    {
      "name": "MyWebScrobbler",
      "type": "webscrobbler", // <-- type is required
      "data": {
        "slug": null,
        "whitelist": [],
        "blacklist": []
      }
    }
  ],
...
```

[MS with Specific file configuration](https://github.com/FoxxMD/multi-scrobbler/blob/master/docsite/docs/configuration/configuration.md#specific-file-configuration)

```
/config
  --> webscrobbler.json

# webscrobbler.json
[
  {
    "name": "MyWebScrobbler",
   // No type required because MS recognizes the file as specific for webscrobbler
    "data": {
      "slug": null,
      "whitelist": [],
      "blacklist": []
    }
  }
]
```

**Comment by FoxxMD** [comment]
@TheFeelTrain thanks for catching the ENV discrepancy. It has been updated to match what it is found documentation and available now on the develop branch/image

**Comment by FoxxMD** [comment]
[Fixed in 0.6.3](https://github.com/FoxxMD/multi-scrobbler/releases/tag/0.6.3)
